tests/jenkins/jobs/PromoteArtifacts_actions_Jenkinsfile

```diff
@@ -0,0 +1,15 @@
+pipeline {
+    agent none
+    stages {
+        stage('promote') {
+            steps {
+                script {
+                    promoteArtifacts(
+                        manifest: 'tests/jenkins/data/opensearch-1.3.0.yml',
+                        fileActions: [createSha512()]
```

Comment:
Assuming we can expand this into `fileActions: [createSha512(), createPgpSig()]`

Reply:
#1352